ci/cloudbuild/builds/cmake-install.sh

```diff
@@ -61,6 +61,7 @@ expected_dirs+=(
   ./include/google/appengine/logging/v1
   ./include/google/cloud/bigquery/logging
   ./include/google/cloud/bigquery/logging/v1
+  ./include/google/cloud/bigquery/v2/minimal/internal
```

Comment:
This is missing `./include/google/cloud/bigquery/v2/minimal`

Reply:
Do I need to add the parent directory ./include/google/cloud/bigquery/v2 as well?

Reply:
Yes, the script is too dumb to figure it out.

Reply:
This build is getting on my nerves...  It does not handle duplicates and `.../v2` was already listed here:

https://github.com/googleapis/google-cloud-cpp/blob/3757a3647ade6a491803c493fe6484285de73526/ci/etc/expected_install_directories#L18

Reply:
Ah...alright. Once more unto... :-)

Reply:
Quoting Henry V just made my day